CMakeLists.txt

```diff
@@ -1,139 +1,195 @@
 # Is this a build as part of Trilinos?
 
+#Pay no attention to the man behind the curtain
```

Comment:
What is this about?

Reply:
Hmm. I had meant to put a real comment here instead of a Wizard of Oz reference. Oops.

Reply:
I will fix this when I updated the README.